### PR TITLE
Battery acid is only in Lead Acid Batteries

### PR DESCRIPTION
**Pull request by tenmillimaster**
#### Summary
Content "Makes Car batteries disassemble-able for acid (and others not)"

#### Purpose of change

For quite a long time, Alkaline/ Carbon zinc batteries were disassemble-able for battery acid, to which later NiMH/NiCD/Li-Ion/LiPo/LiFePo4/Si etc were added. Most of these do not contain sulfuric acid as a working medium, and some even contain materials on the opposite end of the pH scale as their working medium.

#### Describe the solution

Make normal batteries not-dissassemble-able (for now)? Add disassembly recipes for car, motorbike, and small motorbike batteries.

https://www.interstatebatteries.com/-/media/project/interstate-batteries/final-images/technical-documents/safety-data-sheets/lead-acid-battery_us-sds_03-19-2021.pdf
Used above SDS for rough estimates of recoveries. 70% efficiency applied as fudge factor for lead recovery, due to it existing in various forms, and having to be smelted to be re-used.
#### Describe alternatives you've considered

Adding lead sponge item and recipe for further recovery of lead.
Adding lead-oxide & Coke recipe for reduction of lead.
Auditing recipes using Sulfuric acid, and adding an intermediate item, 'battery acid' instead.

#### Testing

Loaded game. Took apart some batteries. Chemically burned feet. Good times.

#### Additional context
PSA DO NOT SMELT BATTERIES irl they have like arsenic and shit in them. Use wheelweights and range lead like a normie.

### CODE REVIEW

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -554,6 +554,30 @@
     "qualities": [ { "id": "CUT", "level": 1 } ],
     "components": [ [ [ "leather", 2 ] ], [ [ "string_6", 2 ] ] ]
   },
+  {
+    "result": "battery_car",
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_deconstruction.json:558]: There is an existing (reversible) crafting recipe for this item, with different components.

File: data/json/recipes/recipe_deconstruction.json [hunk 2]
```diff
@@ -554,6 +554,30 @@
...
+    "type": "uncraft",
+    "activity_level": "LIGHT_EXERCISE",
+    "time": "5 m",
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "components": [ [ [ "chem_sulphuric_acid", 13 ] ], [ [ "lead", 2247 ] ], [ [ "plastic_chunk", 3 ] ] ]
+  },
+  {
+    "result": "battery_motorbike",
```
Comment (anothersimulacrum) [inline data/json/recipes/recipe_deconstruction.json:566]: And this one.

File: data/json/recipes/recipe_deconstruction.json (hunk 1 above)
Comment (anothersimulacrum) [reply data/json/recipes/recipe_deconstruction.json:558]: It would enable you to transmute lead into sulphuric acid, but yes that would work.

Comment (tenmillimaster) [reply data/json/recipes/recipe_deconstruction.json:558]: Ahhh. I see. Gonna go putter around some more with git blame and get the construction/decon values to sit happy with each other then.

**Pull request merged**